target/linux/ramips/dts/mt7620a_sercomm_cpj.dtsi

```diff
@@ -0,0 +1,308 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	aliases {
+		label-mac-device = &ethernet;
+
+		led-boot = &status_green;
+		led-failsafe = &status_amber;
+		led-running = &status_green;
+		led-upgrade = &status_amber;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio1 7 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status_green: led-0 {
+			label = "green:status";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_STATUS;
+		};
+
+		status_amber: led-1 {
+			label = "amber:status";
```

Comment:
Thanks for bump this topic. I'm not sure we should insert such comments because:
1. We should drop them when making backport. So, we always require a manual change after the cherry-pick. Not good.
2. We should add such a comments (or similar, like "has to be converted to the new style after #13837") in each new device support PR.

I suggest making changes to all devices once #13837 is merged.

Reply:
I think separate treatment is reasonable for already existing devices using `label` and no `color`, `function`, which should be converted later, and new devices, which should get rid of `label` ASAP (up to the point of not merging the PR until the patch is merged).

There's also the possibility of dropping `label` now, and having to put with nonfunctioning aliases until the patch is merged.

Reply:
I'm also looking forward to merging https://github.com/openwrt/openwrt/pull/13837. However, I think we shouldn't make that PR a stopper for other activities in the project.

Reply:
Merged now.